src/main/java/org/prebid/server/auction/ExchangeService.java

```diff
@@ -771,7 +788,11 @@ private List<AuctionParticipation> getAuctionParticipation(
             BidderAliases aliases,
             AuctionContext context) {
 
-        final Map<String, JsonNode> bidderToPrebidBidders = bidderToPrebidBidders(bidRequest);
+        final ExtRequestPrebid prebid = extRequestPrebid(bidRequest);
+        final Map<String, JsonNode> bidderToPrebidBidders = bidderToPrebidBidders(
+                prebid == null ? null : prebid.getBidders());
+        final Map<String, JsonNode> bidderToPrebidBidderControls = bidderToPrebidBidders(
+                prebid == null ? null : prebid.getBiddercontrols());
```

Comment:
How about not turning `BIDDER` into `bidder`? This field is for controlling the internal “kitchen” of the auction, and we do not need to modify its contents. In addition, this turning can confuse us later (somewhere in the early stages we will have to use `BIDDER`, and in the later stages `bidder`).

I'm completely against turning it into a `bidder`.

Reply:
P.S.:
https://github.com/prebid/prebid-server-java/blob/bf3fc377f624da419d5b3e849ba83ac04c2f1de1/src/main/java/org/prebid/server/auction/ExchangeService.java#L1309

Be sure to pass in the original `bidderName` and resolve it locally. This is necessary because `biddercontrols` expects the name of the original bidder.